src/github.com/cppforlife/bosh-warden-cpi/vm/warden_creator.go

```diff
@@ -205,6 +205,7 @@ func (c WardenCreator) startAgentInContainer(container wrdn.Container) error {
 				"umount /etc/hostname",
 				"rm -rf /var/vcap/data/sys",
 				"mkdir -p /var/vcap/data/sys",
+				"sed -i 's/chronyc/# chronyc/g' /var/vcap/bosh/bin/sync-time",
```

Comment:
Without any comments added next to the commented-out lines, I'm afraid this would look weird to people inspecting this script. Bosh-Lite is a great platform to learn—I can testify, that's where I've learnt so much about Bosh!

So, another alternative would be to replace the content of the script altogether with a single comment, indicating that the script is a placeholder for syncing time on the Stemcell with Chrony, but here with bosh-lite there is nothing to do in that regards, so `sync-time` the script is emptied instead.

Your thoughts?